CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 0.3 (2021-07-07)
+
+### New
+
+* Added logging (#18) [Natalia Maximo]
+
+* Added pre-commit dev tool (#20) [Natalia Maximo]
+
+### Changes
+
+* Added doc generation pipeline (#17) [Natalia Maximo]
+
+* Reorganized hookspecs (#19) [Natalia Maximo]
+
+### Fix
+
+* Typo in release pipeline file (#23) [Natalia Maximo]
+
+
```

Comment:
The 3.0 docs weren't added when the branch was deployed, so adding this here now to not have any problems with the release process